src/linux.rs

```diff
@@ -110,6 +110,23 @@ impl LinuxI2CDevice {
         Ok(device)
     }
 
+    /// Create a new I2CDevice for the specified path, forcefully setting the
+    /// slave address
+    pub fn force_new<P: AsRef<Path>>(
```

Comment:
This should be marked `unsafe`.

Reply:
Makes sense. What should I add in the docstrings?

Reply:
Something like this: 
```
# Safety

Skips check on whether the address is already in use by a driver.
```